contracts/token/ERC721/ERC721BasicToken.sol

```diff
@@ -4,19 +4,41 @@ import "./ERC721Basic.sol";
 import "./ERC721Receiver.sol";
 import "../../math/SafeMath.sol";
 import "../../AddressUtils.sol";
+import "../../introspection/SupportsInterfaceWithLookup.sol";
 
 
 /**
  * @title ERC721 Non-Fungible Token Standard basic implementation
  * @dev see https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
  */
-contract ERC721BasicToken is ERC721Basic {
+contract ERC721BasicToken is SupportsInterfaceWithLookup, ERC721Basic {
```

Comment:
Unfortunately this is resulting in all of `ERC721BasicToken`'s storage being shifted to accomodate for `SupportsInterfaceWithLookup`'s first.

We could extract the storage variables to a separate contract to fix this. I can't think of any other way.